stream-compositions/api/integrations-api/legal-entity-integration-api/src/main/resources/service-api-v1.yaml

```diff
@@ -0,0 +1,73 @@
+openapi: 3.0.3
+info:
+  title: Legal Entity Integration API
+  version: 1.0.0
+servers:
+  - url: http://localhost:8080/service-api/v1
+    description: The server
+tags:
+  - name: LegalEntityIntegration
+paths:
+  /service-api/v2/legal-entity:
```

Comment:
any specific reason why we are prefixing it with `/service-api/`? what is the purpose of this guy, if it is to provide any external system access it should be `/integration-api/` since we normally use /service-api/ only for interservice communication in between backbase components

Reply:
The main difference between service-api and integration-api are:
* Security (auth0 token vs mTLS)
* use of internal vs external ids.
Because of the triangular architecture, most of the integration endpoints are also available as untrue (because using external ids) service-api endpoints (integration services were not supposed to have any logic, their only purpose was to terminate the mTLS)
When backbase stream was first developed, and triangular architecture was still in place, there were to factors to decide for using service-api endpoints: 
* Not having to bother with mTLS.
* Performance benefit of skipping one service hops.